src/types.cpp

```diff
@@ -510,4 +512,16 @@ ssize_t SizedType::GetAlignment() const
     return 8;
 }
 
+const std::map<std::string, Field> &SizedType::GetStructFields() const
+{
+  assert(IsRecordTy() && record);
+  return record->fields;
+}
+
+const Field &SizedType::GetField(const std::string &name) const
+{
+  assert(IsRecordTy());
+  return record->fields[name];
```

Comment:
`Struct::GetField()` throws exception on missing field (via `.at()`). Do that here too?

Reply:
Nvm, this is fixed in a later commit